CHANGELOG.md

```diff
@@ -52,6 +52,8 @@ These changes are available on the `master` branch, but have not yet been releas
   ([#2448](https://github.com/Pycord-Development/pycord/pull/2448))
 - Fixed missing `application_id` in `Entitlement.delete`.
   ([#2458](https://github.com/Pycord-Development/pycord/pull/2458))
+- Fixed the incorrect type of `RawMemberRemoveEvent.user`.
```

Comment:
This is inconclusive, how about something along the lines of:
```suggestion
- Fixed many inaccurate type hints throughout the library.
```

Reply:
> This is inconclusive, how about something along the lines of:

I think ok, I included it as a more specific description because it was probably the only thing visible from the user's side :P